lib/lightning_web/live/components/credentials.ex

```diff
@@ -376,8 +377,9 @@ defmodule LightningWeb.Components.Credentials do
         <.table id={"#{@id}-table"}>
           <.tr>
             <.th>Name</.th>
+            <.th :if={@show_owner}>Type</.th>
```

Comment:
Shouldn't `:if={@show_owner}` be on the Owner column header ?

Reply:
yes! thanks. fixed.